README.md

```diff
@@ -10,10 +10,10 @@ We will be recreating the control console for the popular Photon laser tag game.
 3. UDP server to send and recieve information over a UDP/IP network to the individual laser tag units.
 4. Game loop to count score and tally up game totals.
 
-# Contributors
-- Thomas Buser 
-- Sophia Forrester
-- Cade DuPont
-- Grace Schmidt
-- Uyen Thi My Ho
-- Vishal Jeyam
\ No newline at end of file
+# Contributors: Real name|GitHub username
+- Thomas Buser | tjbuser
+- Sophia Forrester | 
```

Comment:
I don't know Sophia's username, so I probably need to add it later. 
Her username is asophiaforrester.